src/bonsai/types/summaryTypes.ts

```diff
@@ -151,7 +151,7 @@ export type SubaccountOrder = {
   remainingSize: BigNumber | undefined;
   totalFilled: BigNumber | undefined;
   goodTilBlock: number | undefined;
-  goodTilBlockTime: number | undefined;
+  goodTilBlockTimeMilliseconds: number | undefined;
   createdAtHeight: number | undefined;
```

Comment:
would you be opposed to having both? One in secs and then keeping the Milliseconds?